modules/nwtc-library/src/NWTC_Base.f90

```diff
@@ -67,7 +67,7 @@ MODULE NWTC_Base
 
       INTEGER(C_INTPTR_T)       :: FileAddr  = INT(0,C_INTPTR_T)                   !< The address of file FileName.         (RETURN value from LoadLibrary ) [Windows]
       TYPE(C_PTR)               :: FileAddrX = C_NULL_PTR                          !< The address of file FileName.         (RETURN value from dlopen ) [Linux]
-      TYPE(C_FUNPTR)            :: ProcAddr(NWTC_MAX_DLL_PROC)  = C_NULL_FUNPTR    !< The address of procedure ProcName.    (RETURN value from GetProcAddress or dlsym) [initialized to Null for pack/unpack]
+      TYPE(C_FUNPTR)            :: ProcAddr(NWTC_MAX_DLL_PROC)                     !< The address of procedure ProcName.    (RETURN value from GetProcAddress or dlsym) [initialized to Null for pack/unpack]
```

Comment:
If this `ProcAddr` isn't initialized, does it cause issues with restart capabilities? I wonder if the uninitialized value becomes a problem when we pack/unpack the array like the comment (from a LONG time ago) mentions.

Reply:
I agree that that it would be better to have it initialized. I'll see if it will accept initialization to another value. Unfortunately this compiler thinks that `C_NULL_FUNPTR` isn't valid in this context.

Reply:
Maybe it can be wrapped in a preprocessor directive to be safe? Otherwise, we might need to look at all the time ProcAddr is used and nullify it after the object is created.

I recall adding a safety in the `DLLTypeUnPack` to enforce that these variables are nullified after unpacking if no DLL is used.  https://github.com/OpenFAST/openfast/commit/39d37e6be556ec8e46c721eb23eab67d6d35be6e
But I don't know if such init are needed elsewhere.

Reply:
I found a way to fix this in the latest commit by creating an array of null function pointers.